src/screens/FeedsManager/SupportedChainsCard.tsx

```diff
@@ -537,18 +537,22 @@ export const SupportedChainsCard = withStyles(styles)(
           </ExpansionPanel>
         ))}
 
-        <NewSupportedChainDialog
-          open={newDialogOpen}
-          onClose={() => setNewDialogOpen(false)}
-          onSubmit={handleCreateSubmit}
-        />
-
-        <EditSupportedChainDialog
-          cfg={editCfg}
-          open={isEditDialogOpen()}
-          onClose={handleEditDialogClose}
-          onSubmit={handleUpdateSubmit}
-        />
+        {newDialogOpen && (
```

Comment:
Small optimization here: 
The `NewSupportedChainDialog` and `EditSupportedChainDialog` triggers a lot of api calls if we render them immediately even when the state is close. 

We should only make the api calls when we are actually showing the dialog.